src/openrct2/scenario/Scenario.h

```diff
@@ -162,9 +162,9 @@ struct rct_s6_data
     // SC6[10]
     uint8_t active_research_types;
     uint8_t research_progress_stage;
-    uint32_t last_researched_item_subject;
+    int32_t last_researched_item_subject;
     uint8_t pad_01357CF8[1000];
-    uint32_t next_research_item;
+    int32_t next_research_item;
```

Comment:
How come you decided to go signed? Wouldn't it be better to make a constant for `0xFFFFFFFF` and keep it unsigned?

Reply:
Mostly because the rest of the research items were signed as well. I could have changed all of them, but that would have been more work.